projects/media-viewer/src/lib/store/actions/bookmarks.action.ts

```diff
@@ -1,22 +1,23 @@
 import { Action } from '@ngrx/store';
-import { Bookmark } from '../reducers/bookmarks.reducer';
+import { Bookmark, PdfPosition } from '../../viewers/pdf-viewer/side-bar/bookmarks/bookmarks.interfaces';
 
 export const LOAD_BOOKMARKS = '[Bookmarks] Load Bookmarks';
 export const LOAD_BOOKMARKS_SUCCESS = '[Bookmarks] Load Bookmarks Success';
-export const LOAD_BOOKMARKS_FAIL = '[Bookmarks] Load Bookmarks Failure';
+export const LOAD_BOOKMARKS_FAILURE = '[Bookmarks] Load Bookmarks Failure';
```

Comment:
i think we are using fail everywhere else

Reply:
I've been using failure in bookmarks, but I can change it back if we want it to be consistent everywhere